src/routes/homepage/sections/whyTLIP.svelte

```diff
@@ -1,8 +1,50 @@
 <script lang="ts">
     export let id: string;
+    const HEADINGS = {
+        title: "Why TLIP?",
+        description:
+            "Global trade today has a problem. Paper based trade documentation required for cross border exchange is slow, error prone and expensive."
+    };
+    const REASONS = [
+        {
+            title: "$500 Bn",
+            description: "annual cost of paper based documentation"
+        },
+        {
+            title: "200+",
+            description: "number of information exchanges required for a shipment from Kenya to the Netherlands"
+        },
+        {
+            title: "Zero",
+            description: "the value of a shipment delayed due to lost documents"
+        }
+    ];
 </script>
 
-<section class="container py-72" {id}>
-    <div>WhyTLIP</div>
+<section class=" w-full h-full relative" {id}>
+    <div class="w-full h-full lg:flex">
+        <div class="image w-full h-full lg:w-1/2 bg-cover bg-no-repeat" />
+        <div class="bg-green-1 w-full lg:w-1/2 py-16 lg:py-28 px-6 lg:px-32 text-white">
```

Comment:
They might have changed it recently, `background-color` is now `TLIP Green-500 #005923` which is a little lighter

Reply:
That's right, the color palette has been updated. I'll update it and make the necessary changes :wink: